drivers/wifi/nrfwifi/Kconfig.nrfwifi

```diff
@@ -662,6 +663,66 @@ config NRF_WIFI_RPU_RECOVERY_PROPAGATION_DELAY_MS
 config NET_MGMT_EVENT_QUEUE_SIZE
 	# Doing interface down and up even with delay puts a lot of events in the queue
 	default 16
+
+config NRF_WIFI_RPU_RECOVERY_PROPAGATION_DELAY_MS
+	int "RPU recovery propagation delay in milliseconds"
+	default 10
+	help
+		Propagation delay in milliseconds to wait after RPU is powered down
```

Comment:
nit: Help indentation is inconsistent, typically it's just two extra spaces like below.